fontforge/cvimages.c

```diff
@@ -995,6 +998,7 @@ return(false);
 	if ( endpath==NULL )
     break;
 	start = endpath+1;
+    SCCharChangedUpdate(sc,toback?ly_back:ly_fore);
```

Comment:
Was this previously necessary but missing?

Reply:
Yes